src/pages/ProfilePage/ProfilePage.js

```diff
@@ -585,6 +581,11 @@ const styles = ({breakpoints, palette, spacing, shadows}) => ({
   wrapper: {
     marginBottom: spacing(5),
   },
+  container: {
+    [breakpoints.down('sm')]: {
+      display: 'none',
+    },
+  },
```

Comment:
This probably deserves a comment to explain what is going on here—otherwise it's confusing why you would hide something when the size is small

Reply:
this is for the whole profile container, it should be display=none when the SideBarDrawer is open && the screen = xs. Fixed.